projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -387,17 +387,43 @@ public function comment_form_after() {
 		 */
 		$show_greeting = apply_filters( 'jetpack_comment_form_display_greeting', true );
 
+		/**
+		 * Filter the comment title tag.
+		 *
+		 * @module comments
+		 * @since $$next-version$$
+		 *
+		 * @param string $comment_reply_title_tag The comment title tag. Default to h3.
+		 */
+		$comment_reply_title_tag = apply_filters( 'jetpack_comment_reply_title_tag', 'h3' );
+
 		// The actual iframe (loads comment form from Jetpack server).
 
 		$is_amp = class_exists( Jetpack_AMP_Support::class ) && Jetpack_AMP_Support::is_amp_request();
 		?>
 
 		<div id="respond" class="comment-respond">
-			<?php if ( true === $show_greeting ) : ?>
-				<h3 id="reply-title" class="comment-reply-title"><?php comment_form_title( esc_html( $params['greeting'] ), esc_html( $params['greeting_reply'] ) ); ?>
-					<small><?php cancel_comment_reply_link( esc_html__( 'Cancel reply', 'jetpack' ) ); ?></small>
-				</h3>
-			<?php endif; ?>
+			<?php
+			if ( true === $show_greeting ) :
+				printf(
+					'<%1$s id="reply-title" class="comment-reply-title">',
+					esc_attr( $comment_reply_title_tag )
```

Comment:
Since we're replacing a tag name, not a property value, should this be `esc_html` instead? I tried searching for a bit, but couldn't find the difference between the two besides the filter they are calling. Maybe you know which one is more appropriate here.

Reply:
Both would work here, really, since there is no specific one for tag names. `esc_attr` is appropriate for use inside attributes, `esc_html` will remove all HTML entirely so is good for strings. In this specific case I do not think it really matters, but happy to switch to `esc_html`. 👍

Reply:
Done f3eab8cb21437ea40c3d3ea4e9893a0507025948